tools/jxlinfo.c

```diff
@@ -397,39 +397,65 @@ int PrintBasicInfo(FILE* file, int verbose) {
   return seen_basic_info;
 }
 
+static void print_usage(const char* name) {
+  fprintf(stderr,
+          "Usage: %s [-v] INPUT\n"
+          "  INPUT      input JPEG XL image filename\n"
```

Comment:
```suggestion
          "  INPUT      input JPEG XL image filename(s)\n"
```

Reply:
Ah right, since it now supports multiple inputs.